test/cql-pytest/test_materialized_view.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021-present ScyllaDB
+#
+# This file is part of Scylla.
+#
+# Scylla is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Scylla is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+
+# Tests for materialized views
+
+import time
+import pytest
+
+from util import new_test_table, unique_name
+
+# Test that building an index with a large value succeeds. Regression test
```

Comment:
This test is specific to "view building", i.e., adding a view on a table with pre-existing data. Is the bug specific to view building, or also exists when adding a row to a table which already has a MV enabled?

Reply:
By the way, the reference to "index" is wrong - it's a materialized view in this test, not an index.

Reply:
Actually, the bug is in both paths (regular write and building). The dtest was about view building and that's where I took the inspiration, but it's a good idea to add another case for the view path. It's convenient that on a 1-node cluster each view is local, so I can do away without waiting for eventual consistency.